gradle.properties

```diff
@@ -14,3 +14,5 @@ kotlin.native.cacheKind.linuxX64=none
 kotlin.mpp.hierarchicalStructureSupport=true
 # making custom cinterop libraries available in shared source sets
 kotlin.mpp.enableCInteropCommonization=true
+gprUser=akuleshov7
+gprKey=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
```

Comment:
I guess you already know it, but still -- need to remove it

Reply:
thanks for highlighting my password for everyone :D

Reply:
you are welcome